compiler/env/OMRAOTAdapter.cpp

```diff
@@ -0,0 +1,150 @@
+/*******************************************************************************
```

Comment:
I know the point of this class is to decouple the designs of various things, but it still feels a bit like a "grab bag" class to me just based on the functions that are in it. I'm also concerned that we don't have a better name than "AOT Adapter" which feels too generic to me and therefore indicative of the same problem.

Reply:
Thank you for the comment. Can you please suggest a preferrable name for the class? What can be done to resolve the "grab bag" in your opinion?

Reply:
Thank you for your suggestions. I have renamed the class into ``AOTLoadStoreDriver`` and, hopefully it actually makes things clearer. 

I have trimmed public interface and suggested how clients should use the functions in the comments.